package.json

```diff
@@ -460,8 +461,9 @@
     "jsonc-parser": "^1.0.1",
     "request": "^2.88.0",
     "request-promise": "^4.2.2",
-    "strip-json-comments": "^2.0.1",
     "semver": "^5.6.0",
+    "strip-json-comments": "^2.0.1",
+    "tmp": "0.0.33",
```

Comment:
use ^ to match the latest minor release?

Reply:
^0.0.33 := >=0.0.33 <0.0.34   
So no need to use ^